ui/apps/platform/src/Containers/Navigation/NavigationSideBar/NavigationSideBar.tsx

```diff
@@ -36,10 +35,14 @@ const platformConfigurationPaths = [
     systemHealthPath,
 ];
 
-function NavigationSideBar(): ReactElement {
+type Props = {
```

Comment:
I think we can also add a prefix to this so it'll be `NavigationSideBarProps` (just for consistency with other component prop types)